HLTrigger/Timer/plugins/FastTimerService.cc

```diff
@@ -1374,6 +1388,8 @@ void FastTimerService::writeSummaryJSON(ResourcesPerJob const& data, std::string
   j["modules"].push_back(encodeToJSON("other", "other", data.events, data.overhead));
   j["modules"].push_back(encodeToJSON("eventsetup", "eventsetup", data.events, data.eventsetup));
   j["modules"].push_back(encodeToJSON("idle", "idle", data.events, data.idle));
+  j["modules"].push_back(
+      encodeToJSON(callgraph_.source().moduleName(), callgraph_.source().moduleLabel(), data.events, data.source));
```

Comment:
To keep the same order as before in the JSON files, could you move this right before the other modules, at line 1377 ?